source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -243,9 +267,12 @@ def call_kraken(self, request_type, request, instance, krakens_call):
         """
         logger = logging.getLogger(__name__)
         logger.debug('datetime: %s', request['datetime'])
-        crow_fly_stop_points = set()
+        #We use a dict with a key/value of stop_point.uri and is_zonal
+        #a crow_fly section is added only if stop_point is not zonal
```

Comment:
Since we use three properties on a stop_point : exist, non-odt and odt we can't use a set. You could propose me some variable names?